pkg/inspector/image-inspector_test.go

```diff
@@ -60,12 +61,22 @@ func TestScanImage(t *testing.T) {
 				t.Errorf("%s should have succeeded but failed with %v", k, err)
 			} else {
 				resultFileContent, err := ioutil.ReadFile(v.s.ResultsFileName())
+				if err != nil {
+					t.Errorf("%s should have been able to read the"+
+						"results file but failed with: %v", k, err)
+				}
 				if string(resultFileContent) != string(report) {
-					t.Errorf("%s returned wrong results", k, err)
+					t.Errorf("%s returned wrong results: %v", k, err)
```

Comment:
@pweil- I should have read your previous comment more closely.  I've made the error message more verbose/specific.